wiki/Skinning/osu!taiko/en.md

```diff
@@ -1,474 +1,521 @@
-
-| Animatable   | Beatmap Skinnable |
-|:------------:|:-----------------:|
-| ![No][false] | ![No][false]      |
-
-**Notes:**
-
-- This is an indicator for the spinner.
-
----
-
-![](img/spinner-circle.png "spinner-circle.png")
-
-| Animatable   | Beatmap Skinnable        |
-|:------------:|:------------------------:|
-| ![No][false] | ![No][false] (see notes) |
-
-**Notes:**
-
-- Beatmap skinnable status is suspected to be a bug.
-- For each hit in the spinner happens, the circle rotates anti-clockwise.
-
----
-
-![](img/spinner-approachcircle.png "spinner-approachcircle.png")
-
-| Animatable   | Beatmap Skinnable        |
-|:------------:|:------------------------:|
-| ![No][false] | ![No][false] (see notes) |
-
-**Notes:**
-
-- Beatmap skinnable status is suspected to be a bug.
-- This is the duration indicator of the spinner.
-  - This shrinks over time.
+[true]: /wiki/shared/true.png
+[false]: /wiki/shared/false.png
+
+# osu!taiko
+
+*Main page: [Skinning](/wiki/Skinning).*
+
+*See also: [Skinning osu!taiko Tutorial](/wiki/Skinning_osu!taiko_Tutorial).*
+
+You can override the osu!taiko playfield parts by creating a folder called `taiko` inside your skin folder. If this approach is used, the user will need to explicitly enable this in the [options](/wiki/options) (enable the `Use Taiko skin for Taiko mode` button), otherwise the default skin elements will be used.
+
+## Pippidon
+
+`pippidonclear.png`
+
+![](img/pippidonclear.gif)
+
+| Versions | Animatable | Beatmap Skinnable | Blend Mode | Origin | Suggested SD Size |
+|:-:|:-:|:-:|:-:|:-:|:-:|
+| All | ![Yes][true] (see notes) | ![Yes][true] | Normal | BottomLeft | - |
+
+Notes:
+
+- Animation name: `pippidonclear{n}.png` (without the hyphen (`-`)).
+- The animation rate is BPM dependent.
+- This animation plays when the player hits a combo milestone.
+
+---
+
+`pippidonfail.png`
+
+![](img/pippidonfail.gif)
+
+| Versions | Animatable | Beatmap Skinnable | Blend Mode | Origin | Suggested SD Size |
+|:-:|:-:|:-:|:-:|:-:|:-:|
+| All | ![Yes][true] (see notes) | ![Yes][true] | Normal | BottomLeft | - |
+
+Notes:
+
+- Animation name: `pippidonfail{n}.png` (without the hyphen (`-`)).
+- The animation rate is BPM dependent.
+- This animation plays when the player misses a note or does not have enough health during a break.
+- This overrides `pippidonkiai` if the player misses a note during [kiai time](/wiki/kiai_time).
+
+---
+
+`pippidonidle.png`
+
+![](img/pippidonidle.gif)
+
+| Versions | Animatable | Beatmap Skinnable | Blend Mode | Origin | Suggested SD Size |
+|:-:|:-:|:-:|:-:|:-:|:-:|
+| All | ![Yes][true] (see notes) | ![Yes][true] | Normal | BottomLeft | - |
+
+Notes:
+
+- Animation name: `pippidonidle{n}.png` (without the hyphen (`-`)).
+- The animation rate is BPM dependent.
+- This animation plays when doing nothing (during breaks or waiting for the player to hit the next note)
+
+---
+
+![](img/pippidonkiai.gif "pippidonkiai.png")

```

Comment:
Add the header for `pippidonkiai` for the section and add the proper `pippidonkiai.gif` without the hovertext.